scripts/build.ps1

```diff
@@ -55,13 +55,16 @@ Param(
 
     # Using this option will skip all downloads, if you only need to build launcher
     [switch]$noVendor,
+    
+    # Using this option will specify the emulator to use [all, conemu-maximus5, or windows-terminal]
+    [string]$emulator = 'all',
```

Comment:
@daxgames Would `$terminal` be a more claritive name to use for this switch?

Reply:
Changed to Terminal